src/main/java/com/google/api/generator/spring/SpringWriter.java

```diff
@@ -163,7 +163,7 @@ private static void writeSpringAdditionalMetadataJson(GapicContext context, JarO
     String path = "src/main/resources/META-INF";
     JarEntry jarEntry =
         new JarEntry(String.format("%s/additional-spring-configuration-metadata.json", path));
-    String libName = Utils.getLibName(context);
+    String packageName = context.services().get(0).pakkage();
```

Comment:
nit: Do you mind also extract this to `Utils`? It's only one line, but just in case of future changes.